templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,47 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+<h1>Manufacturer List <a style="margin-left: 20px"
+                         href="{% url 'taxi:manufacturer-create' %}">+</a>
+</h1>
+{% if manufacturer_list %}
+<table class="table">
+  <tr>
+    <th>ID</th>
+    <th>Name</th>
+    <th>Country</th>
+    <th>UPDATE</th>
+    <th>DELETE</th>
+  </tr>
 
-    {% if manufacturer_list %}
-      <table class="table">
-        <tr>
-          <th>ID</th>
-          <th>Name</th>
-          <th>Country</th>
-        </tr>
-
-        {% for manufacturer in manufacturer_list %}
-            <tr>
-              <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
-              </td>
-          </tr>
-        {% endfor %}
-      </table>
-
-    {% else %}
-        <p>There are no manufacturers in the service.</p>
-    {% endif %}
+  {% for manufacturer in manufacturer_list %}
+    <td>
```

Comment:
It seems like the opening <tr> tag is missing for the table row. Each set of <td> elements should be enclosed within <tr> tags to properly define a table row.